docs/guides/circuits.rst

```diff
@@ -0,0 +1,439 @@
+
+**To raise a gate to a power**, you can use ``.power(exponent)`` with the ``exponent`` argument the power you want to raise the gate to. Here we verify that the Hadamard gate raised to the 2nd power is the same as the identity.
+
+.. literalinclude:: /examples/circuits_guide.py
+  :language: python
+  :start-at: ic(H.power(2).matrix == sympy.eye(2))
+  :end-at: ic(H.power(2).matrix == sympy.eye(2))
+
+Where the output is ``ic| H.power(2).matrix == sympy.eye(2): True``
+
+**To get the gate exponential** of a specific gate, you can use ``.exp``, shown in this example with ``.matrix`` to view what the matrix looks like. It is also generally a good idea to simplify the matrix when possible.
+
+.. literalinclude:: /examples/circuits_guide.py
+  :language: python
+  :start-at: ic(sympy.simplify(H.exp.matrix))
+  :end-at: ic(sympy.simplify(H.exp.matrix))
+
+
+.. _wavefunction_operations:
+
+Gate operations vs Wave Function Operations
+-------------------------------------------
+
+While gate operations perform the familiar gates on the qubits of a circuit, `wavefunction operations <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/circuits/_wavefunction_operations.py>`_ can operate on the wavefunction directly. This is convenient if the operation you want to perform is not easily expressible using gates or is not unitary at all. Currently, the only built-in wavefunction operation in Orquestra is the ``MultiPhaseOperation``, which allows a specific phase (given as an angle theta) to be applied to all 2^N components of the wavefunction for an N-qubit circuit.
+
+Wave function operations can be included in the :ref:`creation of a circuit <creating_circuits>`, :ref:`appended later <appending_circuits>`, and :ref:`inspected <inspecting_circuits>` in the same manner GateOperations can be.
+
+.. attention::
+  ``Circuit``\ s containing wave function operations generally cannot be run on backends. Simulators are usually able to.
+
+
+.. _symbolic_gates:
+
+Symbolic gates
+==============
+
+Symbolic gates allow the user to specify parametric gates in terms of parameters that can be set to a specific value/angle later. This allows for a few benefits: 
+
+1. You can create the circuit once and bind the parameters later. Sometimes this saves on computational cost, and it allows for more easily understandable implementations of certain circuits, like the QAOA circuit in this example. 
+2. It gives you a closed formula for the final state of the circuit. This allows users interested in mathematical description and manipulations of the state easier access than could be obtained by varying parameters and running the same circuit multiple times. However, when doing symbolic simulation of the circuit, the performance of the simulation is substantially worse than if the parameters were bound.
```

Comment:
Good place for a shameless plug about `SymbolicSimulator` ;)

Reply:
fixed in af446a1749110a289630084291990551053306a0